misc/snapshotter/config.toml

```diff
@@ -4,13 +4,18 @@ root = "/var/lib/containerd-nydus"
 # The snapshotter's GRPC server socket, containerd will connect to plugin on this socket
 address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
 daemon_mode = "multiple"
-# Snapshotter's debug and trace HTTP server interface
-enable_system_controller = true
 # Whether tp enable stargz support
 enable_stargz = false
 # Whether snapshotter should try to clean up resources when it is closed
 cleanup_on_close = false
 
+[system]
+# Snapshotter's debug and trace HTTP server interface
```

Comment:
Can the interface address be configurated? Should be documented here.

Reply:
Good point. This PR now includes changes that add the config entry to customize the system controller address path